com.unity.render-pipelines.universal/Runtime/UniversalRenderPipeline.cs

```diff
@@ -600,11 +600,49 @@ static void UpdateVolumeFramework(Camera camera, UniversalAdditionalCameraData a
                 UniversalAdditionalCameraData mainAdditionalCameraData = null;
 
                 if (mainCamera != null && mainCamera.TryGetComponent(out mainAdditionalCameraData))
+                {
                     layerMask = mainAdditionalCameraData.volumeLayerMask;
+                }
+
+                trigger = (mainAdditionalCameraData != null && mainAdditionalCameraData.volumeTrigger != null) ? mainAdditionalCameraData.volumeTrigger : trigger;
```

Comment:
Should the `SceneView` camera always update the volume framework? e.g set `shouldUpdate = true` in this else if?

Edit: nvm we only use `shouldUpdate` when we have additional camera data.